### PR TITLE
fix(fe): block editing problem when contest has submission

### PR DESCRIPTION
**Pull request by B0XERCAT**
### Description
image
Contest에 Submission이 존재하는 경우
- import 버튼을 숨깁니다
- Contest Problem List에서 score, order 수정을 막습니다
- score order 필드를 클릭하면 수정할 수 없다는 토스트가 노출됩니다

Closes TAS-1002
### Additional context

---

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md)
- [x] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#pr-and-branch) and follow the [Commit Convention](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#commit-convention)
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

File: apps/frontend/app/admin/contest/[id]/_components/Columns.tsx [hunk 1]
```diff
@@ -68,11 +70,20 @@ export const columns = (
...
+            toast.error('Problem scoring cannot be edited')
+          }
+        }}
+      >
         <Input
+          disabled={disableInput}
           defaultValue={row.getValue('score')}
           className="hide-spin-button w-[70px] focus-visible:ring-0"
           type="number"
+          style={{ pointerEvents: disableInput ? 'none' : 'auto' }}
```
Comment (eunnbi) [inline apps/frontend/app/admin/contest/[id]/_components/Columns.tsx:86]:
```suggestion
          disabled={disableInput}
          defaultValue={row.getValue('score')}
          className="hide-spin-button w-[70px] focus-visible:ring-0 disabled:pointer-events-none"
          type="number"
```

tailwind의 `disabled` 선택자를 사용하면 삼항연산자를 사용하지 않아도 될 것 같아요!

File: apps/frontend/app/admin/contest/[id]/_components/Columns.tsx [hunk 2]
```diff
@@ -117,25 +128,35 @@ export const columns = (
...
+                      : problem
+                  )
                 )
-              )
-            }}
-            className="w-[70px]"
-          />
+              }}
+              className="w-[70px]"
+              disabled={disableInput}
```
Comment (eunnbi) [inline apps/frontend/app/admin/contest/[id]/_components/Columns.tsx:157]:
```suggestion
              className="w-[70px] disabled:pointer-events-none"
```

File: apps/frontend/app/admin/contest/[id]/_components/Columns.tsx [hunk 3]
```diff
@@ -117,25 +128,35 @@ export const columns = (
...
-            options={alphabetArray}
-            onChange={(selectedOrder) => {
-              setProblems((prevProblems: ContestProblem[]) =>
-                prevProblems.map((problem) =>
-                  problem.id === row.original.id
-                    ? { ...problem, order: selectedOrder.charCodeAt(0) - 65 }
-                    : problem
+          }}
+        >
+          <div style={{ pointerEvents: disableInput ? 'none' : 'auto' }}>
```
Comment (eunnbi) [inline apps/frontend/app/admin/contest/[id]/_components/Columns.tsx:139]:
```suggestion
```

Comment (B0XERCAT) [reply apps/frontend/app/admin/contest/[id]/_components/Columns.tsx:139]: 훨씬 깔끔하네요! 감사합니다👍👍

File: apps/frontend/app/admin/contest/[id]/_components/Columns.tsx (hunk 1 above)
Comment (B0XERCAT) [reply apps/frontend/app/admin/contest/[id]/_components/Columns.tsx]: 수정했습니다!

File: apps/frontend/app/admin/contest/[id]/_components/Columns.tsx (hunk 2 above)
Comment (B0XERCAT) [reply apps/frontend/app/admin/contest/[id]/_components/Columns.tsx]: 🐱

**Review by eunnbi — APPROVED** [review]

**Pull request merged**